src/services/VerifiableCredentialService.ts

```diff
@@ -63,7 +63,7 @@ export class VerifiableCredentialService {
   		vc: verifiedCredential,
   	};
 
-  	this.logger.info({ message: "Verified Credential jwt: " }, result);
```

Comment:
The API tests are passing with these changes but we are not seeing any logs about the VC generated to verify the results. Just wondering how do we plan to troubleshoot the VC calculations in prod incase if we had to? Cant there be a way we log the result in an encrypted way using some key so that we could decode the same at our end for verification?

Reply:
I understand that we definitely need to remove any PII from logs.....we can't log VC or the response back from Yoti......for automations testing, Zac would need to clear all messages from queue as part of the framework step and extract the VC from IPV queue for validation.